### PR TITLE
Fixed crash on stopping the server

### PR DESCRIPTION
**Pull request by LogicParrot**

### CODE REVIEW

File: src/World.cpp [hunk 1]
```diff
@@ -215,8 +215,6 @@ cWorld::~cWorld()
 	delete m_LavaSimulator;      m_LavaSimulator     = nullptr;
 	delete m_RedstoneSimulator;  m_RedstoneSimulator = nullptr;
 
-	UnloadUnusedChunks();
```
Comment (LogicParrot) [inline src/World.cpp]: Can anyone confirm that this line is completely unnecessary?

Comment (LogicParrot) [inline src/World.cpp]: - All chunks are already saved
- the chunkmap is about to be reset, which will destroy all chunklayers, which all unload/destroy all chunks. 

Seems redundant to have this line.

Comment (tigerw) [inline src/World.cpp]: It's unnecessary.

**Comment by tigerw** [comment]
For the record, #3115.

**Pull request merged**